build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     ext.ktor_version = '1.1.1'
```

Comment:
Maybe convert to kotlin dsl gradle?

Reply:
Har det noen fordeler over slik det er i dag?